### PR TITLE
Add percent encoding for titles and abstracts when sending metadata to EZID

### PR DESCRIPTION
**Pull request by hardyoyo**
- Add percent encoding for titles and abstracts before sending metadata to EZID
- Add notes on error-handling that needs fixing
- Correct the path used for the POST to EZID (previous posts had an extra slash in them)

### CODE REVIEW

**Comment by myucekul** [comment]
I don't see how this solve the problem EZID has with percent. Has this been tested?

File: plugins/ezid/logic.py [hunk 1]
```diff
@@ -110,8 +110,9 @@ def send_create_request(data, shoulder, username, password, endpoint_url):
         print("%d %s\n" % (ezid_error.code, ezid_error.msg))
         if ezid_error.fp is not None:
             response = ezid_error.fp.read()
-            if not response.endswith("\n"):
-                response += "\n"
+           # FIXME: this code throws errors, endswith doesn't work with response
+           # if not response.endswith("\n"):
+           #     response += "\n"
```
Comment (tingletech) [inline plugins/ezid/logic.py:115]: did you mean to leave this commented out?

Comment (hardyoyo) [reply plugins/ezid/logic.py:115]: Yes, the code throws an error which makes troubleshooting difficult, it's easier with it commented out. I made a card for working the FIXME, It's PUBD-190 (in the backlog)

**Comment by hardyoyo** [comment]
@myucekul yes, this code has been tested and is currently running in production. DOIs (with abstracts) are being minted sucessfully.

**Comment by myucekul** [comment]
Sounds good. Thanks Hardy!

**Pull request merged**